src/develop/blend.c

```diff
@@ -456,7 +456,7 @@ void dt_develop_blend_process(struct dt_iop_module_t *self, struct dt_dev_pixelp
   if(oscale != iscale || xoffs < 0 || yoffs < 0
      || ((xoffs > 0 || yoffs > 0) && (owidth + xoffs > iwidth || oheight + yoffs > iheight)))
   {
-    dt_control_log(_("skipped blending in module '%s': roi's do not match"), self->op);
+    dt_control_log(_("skipped blending in module '%s': ROIs do not match"), self->op);
```

Comment:
As we are at it, since this message is displayed to users we may find some better wording. I doubt a user know what ROI is. Maybe:

`skipped blending in module '%s': working area mismatch`
or
`skipped blending in module '%s': mask area mismatch`

Or anything better :)

Reply:
Indeed, you are right! We shouldn't show users internal terms that aren't even defined in the documentation. Changed to one of your suggestions.